doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -211,7 +211,28 @@ By default, an alert agent will be called for node events, fencing events, and
 resource events. An agent may choose to ignore certain types of events, but
 there is still the overhead of calling it for those events. To eliminate that
 overhead, you may select which types of events the agent should receive.
-   
+
+Alert filters are configured within a ``select`` element inside an ``alert``
+element.
+
+.. list-table:: **Possible alert filters**
+   :class: longtable
+   :widths: 1 3
+   :header-rows: 1
+
+   * - Name
+     - Events alerted
+   * - select_nodes
+     - A node joins or leaves the cluster (whether at the cluster layer for
+       full nodes, or via a remote connection for Pacemaker Remote nodes).
```

Comment:
Do we still use "full nodes" in some contexts for disambiguation, or did we standardize on "cluster nodes"?

Reply:
cluster nodes, thanks